src/server/rdb_load.cc

```diff
@@ -2463,6 +2463,11 @@ void RdbLoader::LoadItemsBuffer(DbIndex db_ind, const ItemsBuf& ib) {
   for (const auto* item : ib) {
     PrimeValue pv;
     if (ec_ = FromOpaque(item->val, &pv); ec_) {
+      if ((*ec_).value() == errc::empty_key) {
+        LOG(ERROR) << "Found empty key: " << item->key << " in DB " << db_ind << " rdb_type "
+                   << item->val.rdb_type;
+        continue;
+      }
       LOG(ERROR) << "Could not load value for key '" << item->key << "' in DB " << db_ind;
```

Comment:
nit: if we already touch this part, let's log ec as well in the "Could not load" part, maybe we'll need it in the future